src/pages/blog/community-day.md

```diff
@@ -0,0 +1,80 @@
+
+## Recognizing our Amazing Contributors
+
+With Astro, we’re privileged to have some amazing project sponsors (more on them in a second). These sponsors allow us to recognize standout members of our community with more than just words. We do this through a program that we call **The Astro Contributor Awards**.
+
+Here’s how it works:
+
+1. Every three months, we open up for nominations in our Discord server.
+2. Anyone can nominate anyone else they think went above & beyond. 
+3. Our **@Core Maintainers** vote on the winners.
+4. **Winners are recognized, celebrated, and awarded 100% of our sponsorship budget for that month (currently at ~$4300).**
+
+This happened to be our biggest month *ever* for nominations, with 36 community nominations received. After voting, we’re excited to announce our first batch of Contributor Award Winners publicly! Our awesome winners are:
+
+- **aFuzzyBear**, for his effort on the community “[XElement](https://github.com/aFuzzyBear/xelement)” project, the “[This Week in Astro](https://www.youtube.com/playlist?list=PLowBAHWxr_2BsoRUzl5uFwQk7-qWxZGts)” video series, and his infectious enthusiasm in our community.
+- **Chris** **(@delucis)**, for significant contributions in both support and documentation. If you’ve noticed the new styling and color on the docs site, you have Chris to thank!
+- **JuanM04**, for significant contributions to the Astro codebase, such as the new `astro add` command.
+- **Lostra**, for significant contributions in community support.
+- **Muhymin**, for significant, continued efforts on the community “[astro-imagetools](https://www.npmjs.com/package/astro-imagetools)” project.
+- **readonlychild**, for significant contributions in community  support.
+- **sarah11918,** for leading the charge on our docs site revamp, joining aFuzzyBear for the “[This Week in Astro](https://www.youtube.com/playlist?list=PLowBAHWxr_2BsoRUzl5uFwQk7-qWxZGts)” video series, and countless other contributions across GitHub and Discord.
+- **ZV**, for significant contributions in both support and documentation.
+
+We’ll be posting this same list in our Discord later today, along with more details and instructions on how to claim your award if you’re one of the people listed here. Congratulations to all of our winners and nominees! 🥳
+
+## Supporting our Essential Dependencies
+
+Some open source projects are lucky, and can raise large amounts of money from sponsors. [Webpack raises $223,113/yr.](https://opencollective.com/webpack) [Babel raises almost $300,000/yr](https://opencollective.com/babel). 
+
+Unfortunately, these types of projects are in a very small minority. For a variety of reasons, most other funded projects receive only a small fraction of this kind of money. 80% of npm’s top 1000 packages received zero funding of any kind (2019). 
```

Comment:
@FredKSchott I removed the footnote here, do we want to link an original source?

Reply:
The problem is I don't really have the original data used to calculate it, this was from a webpage I built back in 2019. it could probably be recreated but I figured the footnote was enough of an answer to not be a problem, even if it wasn't entirely satisfying. Thoughts?

Reply:
Yep I think that's fine!  I removed the footnote thinking it was temporary but I'll add that back in :+1: